Makefile

```diff
@@ -101,12 +102,12 @@ bench:
 .PHONY: testbuild
 testbuild:
 	for p in $(shell $(GO) list $(PKG)); do \
-	  $(GO) test -tags '$(TAGS)' $(GOFLAGS) -c -i $$p || exit $?; \
+	  $(GO) test -cpu $(CPUS) -tags '$(TAGS)' $(GOFLAGS) -c -i $$p || exit $?; \
```

Comment:
I don't think you can specify that at build time since the created binaries have a `-test.cpu` flag (whose default value is probably not influenced by what you pass here).